OpenOversight/app/auth/views.py

```diff
@@ -36,10 +35,12 @@
 )
 from OpenOversight.app.utils.auth import admin_required
 from OpenOversight.app.utils.constants import KEY_APPROVE_REGISTRATIONS
+from OpenOversight.app.utils.flask import sitemap
 from OpenOversight.app.utils.forms import set_dynamic_default
 from OpenOversight.app.utils.general import validate_redirect_url
 
 
+auth = Blueprint("auth", __name__, url_prefix="/auth")
```

Comment:
I feel like the `url_prefix` belongs with the creation of the Blueprint, but I'd be happy to shift it back to the application's `__init__.py` file if everyone else would prefer that.